fiftyone/utils/eta.py

```diff
@@ -753,7 +753,6 @@ def from_keypoint(keypoints):
     return fol.Keypoint(
         label=keypoints.label,
         points=keypoints.points,
-        confidence=keypoints.confidence,
```

Comment:
There's also a `to_keypoints()` method above where you'll need to remove usage of `keypoint.confidence`

Reply:
Oops yeah, let me do that

Reply:
Addressed in the following commit.